hash/msx1_cass.xml

```diff
@@ -143,6 +143,56 @@ license:CC0-1.0
 		</part>
 	</software>
 
+	<software name="svi728nl">
+		<description>Inleiding tot de SpectraVideo SV 728 (Netherlands)</description>
+		<year>1985</year>
+		<publisher>Spectravideo</publisher>
+		<notes>Bundled with Spectravideo SVI-728.</notes>
+		<info name="usage" value="Load with RUN&quot;CAS:&quot;."/>
+		<part name="cass1" interface="msx_cass">
+			<dataarea name="cass" size="44429">
+				<rom name="inleiding tot de spectravideo sv 728 - spectravideo.cas" size="44429" crc="e0d87d84" sha1="f79b1007210a2fad41e56bbb4251af7a4dfa3834"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<!-- Needs a working light pen interface -->
+	<software name="lpengrph" supported="partial">
+		<description>Light Pen Graphic v1.0 (Japan)</description>
+		<year>1984</year>
+		<publisher>Sanyo</publisher>
+		<notes>Bundled with Sanyo MPC-10.</notes>
+		<info name="usage" value="Side A: Load with BLOAD&quot;CAS:&quot;,R. Side B: Load with CLOAD + RUN. Requires a Japanese system."/>
+		<part name="cass1a" interface="msx_cass">
+			<dataarea name="cass" size="26187">
+				<rom name="light pen graphic v1.0 - sanyo (side a).cas" size="26187" crc="cf7c9de7" sha1="4ff9e8bf6a5224819578142a4ba28fa18307a990"/>
+			</dataarea>
+		</part>
+		<part name="cass1b" interface="msx_cass">
+			<dataarea name="cass" size="16800">
+				<rom name="light pen graphic v1.0 - sanyo (side b).cas" size="16800" crc="f8aad1f1" sha1="45a254be32167d2722a50355b673ea39cde74892"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="teachysb">
+		<description>Teach Yourself Basic (United Kingdom)</description>
```

Comment:
“BASIC” is an acronym – it should be in uppercase.

the title of the software was all caps on the manual: https://www.computinghistory.org.uk/det/12628/Teach-Yourself-BASIC/